data-preparation-scripts/generate_calibration_stacs.py

```diff
@@ -208,6 +210,11 @@ def main(arguments: argparse.Namespace) -> None:
         save_calibration_properties_path.parent.mkdir(parents=True, exist_ok=True)
         with open(save_calibration_properties_path, "w") as handle:
             json.dump(calibration_properties_data, handle)
+        count = count + 1
+        if count % 1000 == 0:
+            print(
+                f"I'm alive and have created STACS for {count} of {num_images} - that is {(count/num_images)*100:.2f}%"
```

Comment:
```suggestion
                f"I'm alive and have created STACS for {count} of {num_images} - that is {(count/num_images)*100:.2f}%."
```
Sorry for this one 🙈.